### PR TITLE
macOS setup

### PR DESCRIPTION
**Pull request by zemtsov-artem**

### CODE REVIEW

**Comment by zemtsov-artem** [comment]
@vasily-v-ryabov could you please confirm that there are enough changes for setup.
I saw that we use dev-requirements.txt in AppVeyor.yml and landscape.yml. 
Shall I update them?

File: setup.py [hunk 1]
```diff
@@ -64,10 +64,22 @@ def setup_path(path = ""):
         install_requires.append('pywin32')
 
     packages = ["pywinauto", "pywinauto.tests", "pywinauto.controls", "pywinauto.linux"]
+elif sys.platform == 'darwin':
+    install_requires = [
+        'codecov',
```
Comment (vasily-v-ryabov) [inline setup.py]: `codecov` is a requirement for running tests with coverage (for development only). `setup.py` must contain only production needed packages.

**Comment by vasily-v-ryabov** [comment]
`dev-requirements.txt` contains all packages needed for development of pywinauto. This is not for end users. `setup.py` is for end user installation. There is new feature in `pip` to add `sys.platform` dependent packages into requirements file. Ideally we can modify `dev-requirements.txt` this way.

**Comment by zemtsov-artem** [comment]
> `dev-requirements.txt` contains all packages needed for development of pywinauto. This is not for end users. `setup.py` is for end user installation. There is new feature in `pip` to add `sys.platform` dependent packages into requirements file. Ideally we can modify `dev-requirements.txt` this way.

Applied

File: dev-requirements.txt [hunk 2]
```diff
@@ -1,10 +1,29 @@
-pywin32
-six
+# Win32 specific packages
+#
+pywin32                                   ;sys_platform == 'pywin32'
```
Comment (vasily-v-ryabov) [inline dev-requirements.txt]: Maybe `sys_platform == 'win32'` ?

File: dev-requirements.txt [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+pyobjc-core==5.3                          ; sys_platform == 'darwin'
+pyobjc-framework-AppleScriptObjC==5.3     ; sys_platform == 'darwin'
+pyobjc-framework-ApplicationServices==5.3 ; sys_platform == 'darwin'
+pyobjc-framework-Cocoa==5.3               ; sys_platform == 'darwin'
+pyobjc-framework-CoreText==5.3            ; sys_platform == 'darwin'
+pyobjc-framework-FSEvents==5.3            ; sys_platform == 'darwin'
+pyobjc-framework-Quartz==5.3              ; sys_platform == 'darwin'
+
+# Linux specific packages
+#
```
Comment (vasily-v-ryabov) [inline dev-requirements.txt:16]: `python-xlib` should be mentioned

**Review by vasily-v-ryabov — CHANGES_REQUESTED** [review]

File: dev-requirements.txt [hunk 4]
```diff
@@ -1,10 +1,30 @@
-pywin32
-six
+# Win32 specific packages
+#
+pywin32                                   ;sys_platform == 'win32'
+
+# Darwin specific packages
```
Comment (vasily-v-ryabov) [inline dev-requirements.txt]: `Darwin` -> `macOS`, `Win32` -> `Windows`

**Review by vasily-v-ryabov — CHANGES_REQUESTED** [review]

**Review by vasily-v-ryabov — APPROVED** [review]

**Pull request merged**